public/static/docs/command-reference/list.md

```diff
@@ -0,0 +1,138 @@
+# list
+
+List <abbr>repository</abbr> contents, including files and directories tracked
```

Comment:
Also, does this work for non-Git DVC project? (not technically "repositories"). Checking... ⏳

Reply:
Nope. OK I guess that makes sense you can just `ls` a plain dir.